BackofficeBundle/Resources/config/field_type.yml

```diff
@@ -129,6 +129,18 @@ parameters:
                     default_value: choice
                 input:
                     default_value: array
+        document:
```

Comment:
If you want to embed an another document, for example status, you have to use the same transformer, which is generic, and is supporting 'document'. @nicolasThal